keyboards/kprepublic/bm980hsrgb/keymaps/default/keymap.c

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2021 peepeetee
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [0] = LAYOUT(
+    KC_ESC,   KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,                        KC_PSCR,  KC_SLCK,   KC_PAUS,  KC_DEL,
+	  KC_GRV,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,            KC_NLCK,  KC_PSLS,  KC_PAST,  KC_PMNS,
+	  KC_TAB, 	     KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,       KC_P7,    KC_P8,    KC_P9,    KC_PPLS,
+	  KC_CAPS,          KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_ENT,               KC_P4,    KC_P5,    KC_P6,
+	  KC_LSFT,              KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,                    KC_P1,    KC_P2,    KC_P3,  KC_PENT,
+                                                                                                                                                  KC_UP,
+	  KC_LCTL,    KC_LGUI,    KC_LALT,    KC_SPC,                                                   KC_RALT,  MO(1),     KC_RCTL,                                     KC_P0,    KC_PDOT,
+                                                                                                                                        KC_LEFT,  KC_DOWN,  KC_RGHT
```

Comment:
I'm not sure whether this kind of key ordering (treating keys offset down by 0.25u as a separate row) is ever used by any board in QMK. In fact, most boards just ignore this kind of vertical offset, considering the keys shifted down to be still on the same row. The docs [recommend shifting keys when creating `info.json`](https://docs.qmk.fm/#/reference_configurator_support?id=vertically-offset-keys); however, this may result in slightly wrong coordinates for the LED matrix if you use the current version of my scripts to generate them.

Probably it would be better to swap the keys around in `info.json`, so that you would have that layout part looking like:
```
KC_SLSH,    KC_RSFT,    KC_UP,      KC_P1,    KC_P2,
 KC_RCTL,     KC_LEFT,  KC_DOWN,  KC_RGHT,    KC_P0,
```